evproc/processor.py

```diff
@@ -15,7 +15,8 @@
 
 import stevedore
 
-from evproc import exc
+from collections import deque
+from evproc import exc, Event
```

Comment:
I prefer importing modules and *not* objects, so turn these into:

    import collections
    import evproc
    from evproc import exc